lib/data_update_scripts/20201217184442_append_community_to_community_name.rb

```diff
@@ -0,0 +1,11 @@
+module DataUpdateScripts
+  class AppendCommunityToCommunityName
+    def run
+      return unless SiteConfig.collective_noun_disabled.false?
```

Comment:
```suggestion
      return unless SiteConfig.collective_noun_disabled
```
I think you don't need the `false?` here since `SiteConfig.collective_noun_disabled` returns a boolean and has a default value, so it should _itself_ return `true` or `false`

Reply:
Great ctach! I've updated this

Reply:
Unimportant random comment from the sidelines: the gem we use for `SiteConfig` follows the AR convention of creating predicate methods for boolean options, so this also works and is probably the most idiomatic:

```ruby
return unless SiteConfig.collective_noun_disabled?
```

Reply:
Updated this @citizen428!